util/src/main/java/com/google/cloud/hadoop/util/logging/CustomLogger.java

```diff
@@ -0,0 +1,37 @@
+package com.google.cloud.hadoop.util.logging;
+
+import java.util.Map;
+
+public abstract class CustomLogger {
```

Comment:
If logs will be written to files, may we make `AuditLogger` an instance of `GoogleLoogger` for `com.google.cloud.hadoop.fs.Audit` class and configure it to log to a different audit-log file? I think this is how Hadoop/YARN audit logs are implemented?

Reply:
I wasn't able to find `com.google.cloud.hadoop.fs.Audit` class anywhere. Could you please point me to it? Thanks!

Reply:
I meant that we can create it in GCS connector and use `GoogleLogger` for logging:

```java
package com.google.cloud.hadoop.fs;

public class AuditLogger {
  private static final GoogleLogger logger = GoogleLogger.forEnclosingClass();

  . . .
}
```

For reference, this is what I have found in Hadoop:
- https://github.com/apache/hadoop/blob/0f6ba5e031b93b6a812038edbfa8658a40a5365f/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSNamesystem.java#L8651
- https://github.com/apache/hadoop/blob/trunk/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/NMAuditLogger.java

Reply:
Okay, got for you meant. Thanks for the comments. I will keep these inputs in mind and open the PR against the DataProc repo in the following weeks.